charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -68,59 +68,78 @@ data:
 
       # These values get passed directly into the airflow helm deployments
       helm:
-        # Webserver configuration.
-        webserver:
-          # This is the name of the secret that gets copied from the platform namespace
-          # to the airflow namespaces.
-          jwtSigningCertificateSecretName: {{ template "houston.jwtCertificateSecret" . }}
-          resources:
-            limits:
-              # XXX: There is an alert configured to trigger when ephemeral storage reaches
-              # 1800000000 bytes (90% of the 2Gi limit). Be aware of this when modifying
-              # ephemeral-storage limits.
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # Disable creation of initial user.
-          defaultUser:
-            enabled: false
-    {{- if .Values.global.azure.enabled }}
-          livenessProbe:
-            failureThreshold: 25
-            periodSeconds: 10
-    {{- end }}
-
-        # Worker configuration (applies to Celery and Kubernetes task pods).
-        workers:
-          resources:
-            limits:
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # This is here for upgrading to 0.11.1+ of Airflow.
-          keda:
-            enabled: false
-            namespaceLabels:
-              name: {{ .Release.Name }}
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
-
-        # Scheduler configuration.
-        scheduler:
-
-          # Scheduler pod disruption budget
-          podDisruptionBudget:
-            enabled: false
-
-            # PDB configuration
-            config:
-              maxUnavailable: 1
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
+      {{- if .Values.global.sccEnabled }}
+        # If security context constraints are enabled, enable SCCs for airflow-chart
+        sccEnabled: true
+      {{- end }}
+        airflow:
+          useAstroSecurityManager: true
+          # Webserver configuration.
+          webserver:
+            extraVolumeMounts:
+              - name: signing-certificate
+                mountPath: /etc/airflow/tls
+                readOnly: true
+            extraVolumes:
+              - name: signing-certificate
+                secret:
+                  # This is the name of the secret that gets copied from the platform namespace
+                  # to the airflow namespaces.
+                  secretName: {{ template "houston.jwtCertificateSecret" . }}
+            resources:
+              limits:
+                # XXX: There is an alert configured to trigger when ephemeral storage reaches
+                # 1800000000 bytes (90% of the 2Gi limit). Be aware of this when modifying
```

Comment:
[This was actually changed to 5Gi](https://github.com/astronomer/astronomer/commit/061c85e040d50a849e3ed307e19eed15d53699eb), which means these are out of sync and the alert will not fire at 90% using these defaults. We should update charts/prometheus/templates/prometheus-alerts-configmap.yaml with a comment pointing to this default, and also figure out how we want to reconcile this difference.

Reply:
I think we're going to need to break this out into a separate issue, and skip handling it here. https://github.com/astronomer/issues/issues/3481